2.6.md

```diff
@@ -972,7 +976,7 @@ The presence of `Banner` (Section 3.2.6), `Video` (Section 3.2.7), and/or `Nativ
   <tr>
     <td><code>bidfloorcur</code></td>
     <td>string; default "USD"</td>
-    <td>Currency specified using ISO-4217 alpha codes. This may be different from bid currency returned by bidder if this is allowed by the exchange.</td>
+    <td>Currency specified using ISO-4217 alpha codes. This may be different from bid currency returned by bidder if this is allowed by the exchange. This currency applies to all floors specified in the `Video` or `Audio` objects.</td>
```

Comment:
IMO rephrase the new sentence to something like the following, to future-proof it in case the same case arises in the objects for other formats:
> This currency sets the default for all floors in the message.

Reply:
The trouble is that it doesn't set the currency for all floors in the message. For example, the Deal object has its own bidfloorcur.

Reply:
I suppose we could say "This currency set the default for all floors specified in the `imp` object" or something like that.

Reply:
OK fair -- so today if someone sets `Imp.bidfloorcur` and does *not* set the `Deal.bidfloorcur`, what is the deal's currency? If it's automatically "USD" because that's the default for the missing field, and it does not inherit from `Imp.bidfloorcur`, then this is a minefield, and my strong preference would be to add a currency field in the new object created here.

Reply:
I think the current spec makes it pretty clear that the Deal is transacted in USD if Deal.bidfloorcur is not provided. It talks nothing of inheritance, but it is clear.